tutorials/plotting_tutorial.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
I noticed there are some runtime errors in the saved outputs here—we should rerun the notebook and make sure they don't occur.